common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayLocationV2.scala

```diff
@@ -54,6 +57,9 @@ case class DisplayDigitalLocationV2(
     description =
       "The specific license under which the work in question is released to the public - for example, one of the forms of Creative Commons - if it is a precise license to which a link can be made."
   ) license: Option[DisplayLicenseV2] = None,
+  @Schema(
+    description = "The access conditions"
+  ) accessCondition: Option[DisplayAccessCondition] = None,
```

Comment:
Jonathan’s examples in https://github.com/wellcometrust/platform/issues/4125 and https://github.com/wellcometrust/platform/issues/3946 are both lists – should this be

```scala
) accessCondition: Option[List[DisplayAccessCondition]] = None
```

?